### PR TITLE
layout redesign

### PR DESCRIPTION
**Pull request by r0mille**
let me know what you think! if there are too many words or something.

### CODE REVIEW

File: components/nav.tsx [hunk 1]
```diff
@@ -55,40 +55,40 @@ export default function Nav({}) {
...
           </a>
           <a
-            href="#reveal"
+            href="#the-santas"
             className="block mt-4 lg:inline-block lg:mt-0 text-teal-200 hover:text-white mr-4 cursor-pointer"
           >
-            OPEN presents
+            TEAM
           </a>
+          <s className="mr-4">FEEL the love</s>
```
Comment (brock-haugen) [inline components/nav.tsx:82]: I'd say we just drop this one 🤷‍♀️

File: components/nav.tsx [hunk 2]
```diff
@@ -103,7 +103,7 @@ export default function Nav({}) {
             </a>
           </span>
           <span className="mx-5">
-            <a href="https://opensea.io" target="_blank">
+            <a href="https://opensea." target="_blank">
```
Comment (brock-haugen) [inline components/nav.tsx:106]: should still be `.io` right?

File: pages/_app.tsx [hunk 3]
```diff
@@ -41,7 +41,7 @@ export default function App({ Component, pageProps }) {
         }
 
         .alt-font {
-          font-family: "Special Elite", cursive;
+          font-family: "Iniya";
```
Comment (brock-haugen) [inline pages/_app.tsx:44]: I like this font 👌

File: components/how-to.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        <p className="text-2xl font-extralight	">
+          To mint your own Raging Santa you must provide a gift NFT of your
+          choice for the Raging Santas Gift Pool.
+          <br />
+          <br />
+          Each Raging Santa costs 0.03 ETH + gas to mint.
+          <br />
+          <br />
+          Your Raging Santa will be proof that you gave generously to the Gift
+          Pool and will also be proof of your claim to a Free Gift on X-Mas Day!
```
Comment (brock-haugen) [inline components/how-to.tsx:17]: I think we should be consistent on "X-Mas" or "Christmas" 🤔 maybe "Christmas"?

**Review by brock-haugen — APPROVED** [review]

File: components/nav.tsx (hunk 2 above)
Comment (r0mille) [reply components/nav.tsx:106]: oh weird - yah didnt mean to delete

File: components/nav.tsx (hunk 1 above)
Comment (r0mille) [reply components/nav.tsx:82]: ok! yeah we're sorta dropping the snarky old-timey ad-speak so i will remove

File: components/how-to.tsx (hunk 4 above)
Comment (r0mille) [reply components/how-to.tsx:17]: good call - i like XMas bc its shorter and it has an X. but down for whatever

**Pull request merged**